src/index.tsx

```diff
@@ -106,8 +111,10 @@ export const useKeyboardShortcuts = (
     if (event instanceof KeyboardEvent) {
       const keys = shortcut.keys.filter(withoutComboKeys)
       const valid = validateKeys(keys)
+      const singleKey =
+        shortcut.keys.length === 1 && shortcut.keys[0] === event.key
 
-      if (!valid || getKeyCode(keys[0]) !== event.code) return
+      if (!valid || !(singleKey || getKeyCode(keys[0]) === event.code)) return
```

Comment:
I think you can remove the `( )` around `(singleKey || getKeyCode(keys[0]) === event.code)`

Reply:
It is intentional. I think it's easier to digest 

```ts
!(singleKey || getKeyCode(keys[0]) === event.code)
```

Than

```ts
!singleKey && getKeyCode(keys[0]) !== event.code
```